### DISCUSSION TITLE
API for restream

### DISCUSSION THREAD

**Discussion by alexlealgalvao** [datarhei/restreamer#577] [General]
Friends, how are you?
Has anyone developed a client that consumes the API?

**Comment by svenerbeck** [comment]
Hello alexlealgalvao,

We're doing well, thank you for asking!

To address your question about a client consuming the API: Yes, the Restreamer is essentially a client of our API. If you're interested in understanding how it interacts with the API or need details for your development, refer to our API documentation at [https://docs.datarhei.com/core](https://docs.datarhei.com/core).

Feel free to reach out if you have any questions or need further guidance!

Best regards,
